basic/main.go

```diff
@@ -0,0 +1,30 @@
+// basic is a program used to define a goroutine leak.
```

Comment:
I like this code now. It is clear and shows a real leak. Beginners will make these types of mistakes. Let's just clean up the fmt calls and that one comment below.